src/OSPSuite.Core/Comparison/CalculationMethodCacheDiffBuilder.cs

```diff
@@ -0,0 +1,20 @@
+﻿using OSPSuite.Assets;
+using OSPSuite.Core.Domain;
+
+namespace OSPSuite.Core.Comparison
+{
+   public class CalculationMethodCacheDiffBuilder : DiffBuilder<CalculationMethodCache>
+   {
+      private readonly EnumerableComparer _enumerableComparer;
+
+      public CalculationMethodCacheDiffBuilder(EnumerableComparer enumerableComparer)
+      {
+         _enumerableComparer = enumerableComparer;
+      }
+
+      public override void Compare(IComparison<CalculationMethodCache> comparison)
+      {
+         _enumerableComparer.CompareEnumerables(comparison, x => x, item => item.Category, missingItemType: Captions.CalculationMethod);
```

Comment:
Identity is per category as we should have on entry per category. Category missing will be displayed as present absent